packages/web-components/src/components/cbp-notice/cbp-notice.scss

```diff
@@ -0,0 +1,60 @@
+/**
+ * @prop --cbp-list-padding: var(--cbp-space-4x);
+*/
+
+:root {  
+  --cbp-notice-background: var(--cbp-color-info-lighter);
+  --cbp-notice-background-dark: var(--cbp-color-info-dark);
+  --cbp-notice-border-left-color: var(--cbp-color-info-base);
+  --cbp-notice-border-left-color-dark: var(--cbp-color-info-light);
+}
+
+
+[data-cbp-theme=light] cbp-notice[context*=dark]:not([context=light-always]),
+[data-cbp-theme=dark] cbp-notice:not([context=dark-inverts]):not([context=light-always]) {
+  --cbp-notice-background: var(--cbp-notice-background-dark);
+  --cbp-notice-border-left-color: var(--cbp-notice-border-left-color-dark);
+}
+
+cbp-notice {
+  display: block;
+
+
+  .notice-container{
```

Comment:
I don't think you need this wrapper. Your component is the wrapper.

Reply:
removed